v4/source/users.yaml

```diff
@@ -1093,6 +1369,26 @@
             userID := "4xp9fdt77pncbef59f4k1qe83o"
 
             ok, resp := Client.SetProfileImage(userID, data)
+        - lang: PHP
+          source: |
+            require 'vendor/autoload.php';
+
+            use \Gnello\Mattermost\Driver;
+
+            $container = new \Pimple\Container([
+                "driver" => [
+                    "url" => "https://your-mattermost-url.com",
+                    "login_id" => "email@domain.com",
+                    "password" => "Password1",
+                  ]
+            ]);
+
+            $driver = new Driver($container);
+            $driver->authenticate();
+
+            $result = $driver->getUserModel()->setUserProfileImage("userid", [
+                "image" => fopen("profile_pic.png", 'rb'),
```

Comment:
If I am not mistaken, here we are passing just the file identifier. Will this behave as expected?

Reply:
Yes, you're right but in the end it actually sends the binary file, as it mentioned [here](http://docs.guzzlephp.org/en/latest/quickstart.html#sending-form-files) in the http client documentation used by the php client. 
Anyway this "magic" is handled by the http client and I agree that it could be more clearer to provide a StreamInterface instead.